core/ops_builtin_v8.rs

```diff
@@ -712,13 +712,25 @@ fn op_destructure_error(
   JsError::from_v8_exception(scope, error.v8_value)
 }
 
+/// Effectively throw an uncatchable error. This will terminate runtime
+/// execution before any more JS code can run, except in the REPL where it
+/// should just output the error to the console.
 #[op(v8)]
-fn op_terminate(scope: &mut v8::HandleScope, exception: serde_v8::Value) {
+fn op_dispatch_exception(
```

Comment:
I renamed this because it no longer terminates in every case. Now it just represents unhandled error events.